cypress-tests/cypress/support/pageBuilder/pbUpdatingPageTemplate.ts

```diff
@@ -0,0 +1,70 @@
+import { gqlClient } from "../utils";
+
+declare global {
+    // eslint-disable-next-line @typescript-eslint/no-namespace
+    namespace Cypress {
+        interface Chainable {
+            pbUpdatingPageTemplate(id: string, data: any): Promise<any>;
+        }
+    }
+}
+
+const MUTATION = /* GraphQL */ `
+    mutation UpdatePageTemplate($id: ID!, $data: PbUpdatePageTemplateInput!) {
+        pageBuilder {
+            pageTemplate: updatePageTemplate(id: $id, data: $data) {
+                data {
+                    id
+                    title
+                    slug
+                    tags
```

Comment:
Let's remove unneeded fields. Once we do that, let's change `Promise<any` in L7 to `Promise<{ FIELDS_THAT_REMAINED IN THE LIST }>`, e.g.:

```ts
Promise<{id: string, title: string, slug: string}>
```